src/features/common/Layout/UserLayout/UserLayout.module.scss

```diff
@@ -40,6 +40,15 @@
 .profilePageWrapper {
   width: 100%;
   min-height: 100vh;
+  .widgetsContainer {
+    padding: 0px; 
+    margin-top: 130px;
+  }
+    @include xsPhoneView {    
+      & .widgetsContainer {
+          padding-top: 80px !important;
```

Comment:
Not sure if this is necessary testing with Chrome some mobile device views:

www1.plant-for-the-planet.org:

<img width="355" alt="Bildschirmfoto 2023-07-27 um 09 18 13" src="https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/1532418/f1a994df-dc51-487d-98ad-9af3e523a12f">


After your change having some additional white space:

<img width="353" alt="Bildschirmfoto 2023-07-27 um 09 17 51" src="https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/1532418/01d50fbe-2428-457f-942f-7f71be5aab0d">